bot/tests/processors/test_pennychat.py

```diff
@@ -77,6 +84,69 @@ def test_time_select(mocker):
     }
 
     bot_module(event)
-    penny_chat = PennyChat.objects.get(id=chat_id)
+    penny_chat = PennyChatInvitation.objects.get(id=chat_id)
     test_time = datetime.now().replace(hour=12, minute=0, second=0, microsecond=0).time()
     assert penny_chat.date.astimezone(TZ).time() == test_time
+
+
+@pytest.mark.django_db
+def test_PennyChatBotModule_share(mocker):
+    organizer = UserProfile.objects.create(slack_id='organizer')
+    user_invitee_1 = UserProfile.objects.create(slack_id='invitee')
+    # make sure that things don't break if for some reason a user attempts to invite themselves
+    user_invitee_2 = organizer
+
+    penny_chat_invitation = PennyChatInvitation.objects.create(
+        invitees=','.join([user_invitee_1.slack_id, user_invitee_2.slack_id]),
+        user=organizer.slack_id,
+        date=timezone("America/Los_Angeles").localize(datetime(1979, 10, 12)),
+        title='fake title',
+        description='fake description',
+    )
+
+    event = {
+        'user': {
+            'id': organizer.slack_id
+        },
+        'trigger_id': 'hi there',
+        'actions': [{'action_id': 'penny_chat_share'}],
+        'response_url': 'http://some_website.com',
+    }
+
+    def ids_mock(user_ids, slack_client):
+        lookup = {
+            organizer.slack_id: organizer,
+            user_invitee_1.slack_id: user_invitee_1,
+        }
+        return {user_id: lookup[user_id] for user_id in user_ids}
+
+    def id_mock(user_id, slack_client):
+        return ids_mock([user_id], slack_client).get(user_id)
+
+    with mocker.patch('bot.processors.pennychat.get_or_create_user_profile_from_slack_ids', side_effect=ids_mock), \
+            mocker.patch('bot.processors.pennychat.get_or_create_user_profile_from_slack_id', side_effect=id_mock), \
+            mocker.patch('bot.processors.pennychat.requests'):
+
+        # The Actual Test
+        PennyChatBotModule(mocker.Mock()).share(event)
+
```

Comment:
if you share the event, then that's when we create the actually penny chat and participants